js/packages/components/main/Share.tsx

```diff
@@ -82,26 +83,30 @@ const ScanBody: FC<{ visible: boolean }> = ({ visible = true }) => {
 				flex.justify.center,
 				border.radius.medium,
 				{
+					overflow: 'hidden',
 					height: qrScanSize,
 					aspectRatio: 1,
+					position: 'relative',
 				},
 			]}
 		>
 			{visible && (
-				<QRCodeScanner
-					onRead={({ data, type }) => {
-						if ((type as string) === 'QR_CODE' || (type as string) === 'org.iso.QRCode') {
+				<Camera
+					onBarCodeScanned={({ data, type }) => {
+						console.log({ type })
+						if ((type as string) === 'qr') {
```

Comment:
A qr constant that define 'qr' as string doesn't exist in `expo-camera` ? or `expo-barcode-scanner` ?